terraform/environments/digital-prison-reporting/main.tf

```diff
@@ -97,36 +97,43 @@ module "glue_demo_table" {
   glue_table_depends_on = [module.glue_database.db_name]
 }
 
-# Glue Cloud Platform ETL JOB
+# Glue Cloud Platform Ingestion Job (Load, Reload, CDC)
 module "glue_cloudplatform_reporting_job" {
   source                        = "./modules/glue_job"
   create_job                    = local.create_job
   name                          = "${local.project}-reporting-hub-${local.env}"
   description                   = local.description
+  glue_version					= 4.0
+  command_type					= "gluestreaming"
   create_security_configuration = local.create_sec_conf
-  job_language                  = "scala"
   temp_dir                      = "s3://${module.s3_glue_jobs_bucket[0].bucket.id}/tmp/reporting-hub/"
   checkpoint_dir                = "s3://${module.s3_glue_jobs_bucket[0].bucket.id}/checkpoint/reporting-hub/"
   spark_event_logs              = "s3://${module.s3_glue_jobs_bucket[0].bucket.id}/spark-logs/reporting-hub/"
   tags                          = local.all_tags
-  script_location               = "s3://${local.project}-artifact-store-${local.environment}/artifacts/cloud-platform/digital-prison-reporting-poc/cloud-platform-reporting-hub-vLatest.scala"
```

Comment:
we need the "script_location" but new value ?

Reply:
There isn't a script in the new jobs. It is mandatory in the UI so we put a valid bucket but the path isn't needed.